src/frontend/main.go

```diff
@@ -94,6 +94,13 @@ func main() {
 	}
 	log.Out = os.Stdout
 
+	prometheusExporter, err := prometheus.NewExporter(prometheus.Options{})
```

Comment:
We might want to call `err` something else? It doesn't feel great to use a generic term like this within the `main` func

Reply:
err is kind of the default name in go. Yeah, golang exception handling is a world on its own...